rotkehlchen/chain/bitcoin/bch/manager.py

```diff
@@ -26,13 +40,36 @@ def __init__(self, database: 'DBHandler') -> None:
         super().__init__(
             database=database,
             blockchain=SupportedBlockchain.BITCOIN_CASH,
+            asset=A_BCH,
+            event_identifier_prefix=BCH_EVENT_IDENTIFIER_PREFIX,
             api_callbacks=[BtcApiCallback(
                 name='haskoin',
                 balances_fn=self._query_haskoin_balances,
                 has_transactions_fn=self._query_haskoin_has_transactions,
-                transactions_fn=None,  # TODO: add support for querying transactions
+                transactions_fn=self._query_haskoin_transactions,
             )],  # TODO: add other APIs for fallbacks if haskoin goes down
         )
+        self.converted_addresses: dict[BTCAddress, BTCAddress] = {}
+
+    def refresh_tracked_accounts(self) -> None:
+        """Refresh the tracked accounts and ensure the CashAddr format of each address is used
+        since all addresses returned from haskoin are in the CashAddr format.
+        """
+        super().refresh_tracked_accounts()
+        self.converted_addresses.clear()
+        for i, account in enumerate(self.tracked_accounts):
```

Comment:
nitpick/ we dont use `i`

```suggestion
        for idx, account in enumerate(self.tracked_accounts):
```